src/server.rs

```diff
@@ -115,9 +115,11 @@ impl Service for Server {
         // from http://jaketrent.com/post/https-redirect-node-heroku/
         if let Some(raw) = req.headers().get_raw("x-forwarded-proto") {
             if raw != &b"https"[..] {
+                let host: &Host = req.headers().get().unwrap();
```

Comment:
I like the latter, `let host = req.headers().get::<Option<&Host>>().unwrap();` because it helps define how `get()` acts to retrieve that header field's value. I will update this PR. :+1: